shard.yml

```diff
@@ -9,16 +8,13 @@ authors:
 dependencies:
   active-model:
     github: spider-gazelle/active-model
-    version: ~> 1.8
```

Comment:
why did we get rid of these version pins?

Reply:
Was getting frustrated at shards not resolving, so went nuclear on it all.
Safe to add them back to the latest `~>` versions for the next version bump now everything is functional